python/tensorrt_model_connect/families/parakeet_tdt/fastconformer.py

```
@@ -0,0 +1,648 @@
+                    "unk_id": sp.unk_id(),
+                    "vocab": [[piece, 0.0] for piece in vocab],
+                },
+                "added_tokens": [],
+                "normalizer": None,
+                "pre_tokenizer": None,
+                "post_processor": None,
+                "decoder": {"type": "Metaspace", "replacement": "\u2581", "add_prefix_space": True},
+            }
+            tok_json_path.write_text(json.dumps(tok_json))
+        except Exception:
+            pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)